app.py

```diff
@@ -34,9 +34,9 @@ def init_app(self):
 
         # Register a menu entry on the ShotGrid menu so that users can launch the panel.
         self.engine.register_command(
-            "Scene Breakdown...",
+            "{}...".format(self.get_setting("display_name")),
             self.create_panel,
-            {"type": "panel", "short_name": "breakdown"},
+            {"type": "panel", "short_name": "breakdown2"},
```

Comment:
This should be breakdown to resolve the unit tests

```suggestion
            {"type": "panel", "short_name": "breakdown"},
```